packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -334,19 +334,29 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
       theme,
       title,
       keytipProps,
-      placeholder,
+      placeholder: placeholderProp,
       tabIndex,
       autofill,
       persistMenu,
-      iconButtonProps
+      iconButtonProps,
+      multiSelect
     } = this.props;
     const { isOpen, focused, suggestedDisplayValue } = this.state;
     this._currentVisibleValue = this._getVisibleValue();
+    const multiselectPlaceholder = multiSelect
+      ? this._getMultiselectDisplayString(this.state.selectedIndices, this.state.currentOptions, suggestedDisplayValue)
+      : '';
```

Comment:
>''; [](start = 8, length = 3)

Should this be set to `undefined` instead of an empty string?

Reply:
Maybe. I'll change it and see.